slides/02-data-budget.qmd

```diff
@@ -224,56 +233,57 @@ We will use this tomorrow
 
 ![](images/parsnip-flagger.jpg){.absolute top="0" right="0" width="150" height="150"}
 
-*Explore the `frog_train` data on your own!*
+*Explore the `taxi_train` data on your own!*
 
-* *What's the distribution of the outcome, latency?*
-* *What's the distribution of numeric variables like age?*
-* *How does latency differ across the categorical variables?*
+* *What's the distribution of the outcome, tip?*
+* *What's the distribution of numeric variables like distance?*
+* *How does tip differ across the categorical variables?*
 
 ```{r}
 #| echo: false
-countdown(minutes = 8, id = "explore-frogs")
+countdown(minutes = 8, id = "explore-taxi")
 ```
 
 ::: notes
 Make a plot or summary and then share with neighbor
 :::
 
+
+:::notes
+
+:::
+
 ## 
 
 ```{r}
 #| fig-align: 'center'
-ggplot(frog_train, aes(latency)) +
-  geom_histogram(bins = 20)
+ggplot(taxi_train, aes(x = local, fill = tip)) +
```

Comment:
I had trouble finding bi/trivariate visualizations that felt very informative. Feedback here very much appreciated!

Reply:
I would keep this plot as the one looking at the distribution of the response, also because it's the first prompt for the exercise

Reply:
So a univariate `geom_bar()` for this first one?

Reply:
[0b088c5](https://github.com/tidymodels/workshops/pull/108/commits/0b088c5fcbef1f794fbdcfb3f7201786bdf99a39) :)